cv/qc-cv.obo

```diff
@@ -539,14 +564,1316 @@ is_a: QC:4000009 ! ID based
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+property_value: has_units UO:0000189
+
+[Term]
+id: QC:4000138
+name: MZ acquisition range
+def: "Upper and lower limit of m/z values at which spectra are recorded." [PSI:QC]
+comment: Acquisition levels can be used as a criterion to assess the comparability of instrument settings between runs. 
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000004 ! n-tuple
+property_value: has_units MS:1000040
+
+[Term]
+id: QC:4000139
+name: RT acquisition range
+def: "Upper and lower limit of time at which spectra are recorded." [PSI:QC]
+comment: Acquisition levels can be used as a criterion to assess the comparability of instrument settings between runs. 
+is_a: QC:4000004 ! n-tuple
+property_value: has_units UO:0000010
+
+[Term]
+id: QC:4000140
+name: Fastest frequency for MS level 1 collection
+def: "Fastest frequency for MS level 1 collection" [PSI:QC]
+comment: Spectrum acquisition frequency can be used to gauge the suitability of used instrument settings for the sample content used.
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+property_value: has_units UO:0000106
+
+[Term]
+id: QC:4000141
+name: Fastest frequency for MS level 2 collection
+def: "Fastest frequency for MS level 2 collection" [PSI:QC]
+comment: Spectrum acquisition frequency can be used to gauge the suitability of used instrument settings for the sample content used.
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+property_value: has_units UO:0000106
+
+[Term]
+id: QC:4000142
+name: Slowest frequency for MS level 1 collection
+def: "Slowest frequency for MS level 1 collection" [PSI:QC]
+comment: Spectrum acquisition frequency can be used to gauge the suitability of used instrument settings for the sample content used.
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+property_value: has_units UO:0000106
+
+[Term]
+id: QC:4000143
+name: Slowest frequency for MS level 2 collection
+def: "Slowest frequency for MS level 2 collection" [PSI:QC]
+comment: Spectrum acquisition frequency can be used to gauge the suitability of used instrument settings for the sample content used.
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+property_value: has_units UO:0000106
+
+[Term]
+id: QC:4000144
+name: Precursor intensity range
+def: "Minimum and maximum precursor intensity recorded." [PSI:QC]
+comment: The intensity range of the precursors informs about the dynamic range of the acquisition. 
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000004 ! n-tuple
+property_value: has_units MS:1003085
+
+[Term]
+id: QC:4000145
+name: MS1 frequency in RT quarters
+def: "MS1 frequency in RT 1st quarter, 2nd quarter, 3rd quarter, 4th quarter" [PSI:QC]
```

Comment:
so quarter != quantile, right?
This is a bit confusing why quarter is used here.
Is this implemented somewhere?